src/Drivers/CMakeLists.txt

```diff
@@ -13,12 +13,18 @@ target_link_libraries(nlpDenseCons_ex3.exe hiop)
 add_executable(nlpMDS_ex4.exe nlpMDS_ex4_driver.cpp)
 target_link_libraries(nlpMDS_ex4.exe hiop)
 
+add_executable(nlpMDS_ex6.exe nlpMDS_ex6_driver.cpp)
+target_link_libraries(nlpMDS_ex6.exe hiop)
+
 if(HIOP_USE_RAJA)
   if(HIOP_USE_GPU)
     set_source_files_properties(nlpMDSForm_raja_ex4.cpp nlpMDS_ex4_raja_driver.cpp PROPERTIES LANGUAGE CUDA)
+    set_source_files_properties(nlpMDSForm_raja_ex6.cpp nlpMDS_ex6_raja_driver.cpp PROPERTIES LANGUAGE CUDA)
   endif()
   add_executable(nlpMDS_ex4_raja.exe nlpMDS_ex4_raja_driver.cpp nlpMDSForm_raja_ex4.cpp)
   target_link_libraries(nlpMDS_ex4_raja.exe hiop)
+  add_executable(nlpMDS_ex6_raja.exe nlpMDS_ex6_raja_driver.cpp nlpMDSForm_raja_ex6.cpp)
```

Comment:
I think we should add MDS Ex6 to the suite of tests

Reply:
Perhaps we should think about subdirectory structure in `src/Drivers`? It is getting crowded there.

Reply:
We could move current `tests/` directory to `tests/unit/` and move `src/Drivers` to `tests/functionality`?

Reply:
Actually, my suggestion is out of scope for this PR.

Reply:
@ashermancinelli That structure of `tests/` looks good, but I prefer `Drivers/` to stay in the current location and have the current content as they are user oriented, while `tests/` not so much. In `tests/functionality` we can add tests additional tests based on `src/Drivers/`.